piet/src/samples/picture_16.rs

```diff
@@ -0,0 +1,47 @@
+//! capture_image_rect
+//!
+//! This tests makes sure that copying part of an image works
+
+use crate::kurbo::{Rect, Size};
+use crate::{Color, Error, InterpolationMode, RenderContext};
+
+pub const SIZE: Size = Size::new(200., 200.);
+
+const RED: Color = Color::rgb8(255, 0, 0);
+const BLUE: Color = Color::rgb8(0, 0, 255);
+const INTERPOLATION_MODE: InterpolationMode = InterpolationMode::NearestNeighbor;
+const BORDER_WIDTH: f64 = 2.0;
```

Comment:
Let's increase `BORDER_WIDTH` to `4.0`. The reasoning being that we will shrink the 160x160 box down to four 40x40 boxes for the corners, which means that we need four fixels for every pixel to accurately match the downsize. Fewer pixels might work in some cases, but `4.0` needs to definitely work with all implementations so it is a good test value.